cli/flags.rs

```diff
@@ -1000,8 +1000,14 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
       Arg::with_name("fail-fast")
         .long("fail-fast")
         .alias("failfast")
-        .help("Stop on first error")
-        .takes_value(false),
+        .help("Exit the test suite immediately upon n number of failures, By default stop on first error")
```

Comment:
```suggestion
        .help("Exit the test suite immediately upon n number of failures, by default stop on first error")
```

Reply:
I think we need to go with @caspervonb suggestion something like `Stop after N errors`.